assets/externalized/vehicles.json

```diff
@@ -0,0 +1,72 @@
+/**
+ * Definitions of player-controllable transport vehicles.
+ *
+ * The ordering in this list is important, and the game has hard-coded references to the
+ * vehicles. For example, the game expects the HELICOPTER to be the 6th vehicle.
+ *
+ * In base vanilla game, only the Hummer, Ice-cream truck and Helicopter are available.
+ *
+ * Fields:
+ * - enterSound: Sound to play when a soldier enters the vehicle. The value must be a valid SoundID.
+ * - moveSound: Sound to play when the vehicle moves. The value must be a valid SoundID.
+ * - profileID: Profile ID associated to the vehicle.
+ * - movementType: Determines if the vehicle can travel through map sectors and how fast. Must be one of FOOT, CAR, TRUCK, TRACKED or AIR.
+ * - armourType: Determines the vehicle's protection against damage. Use the internal name of any armour items.
+ * - seats: Maximum number of soldiers allowed in the vehicle. Must not be greater than 10.
+ */
+[
+    // Eldorado (unused)
+    {
+        "enterSound": 85, // S_VECH1_INTO
```

Comment:
Can we just write the sound string here? Other wise we limit mods to use the sound files that are included with the game.

Reply:
It's also done that way in `weapons.json`:

```
"Sound": "sounds/weapons/9mm single shot.wav",
```

Reply:
Updated. Thanks!